I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 252, in script call
    call setup_icons()
  File "game/script.rpy", line 44, in script
    call screen Match_Three
ScriptError: could not find label 'win_screen'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\Corv1122\Downloads\Ren'Py\renpy-8.3.4-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\Users\Corv1122\Downloads\Ren'Py\renpy-8.3.4-sdk\renpy\main.py", line 652, in main
    run(restart)
  File "C:\Users\Corv1122\Downloads\Ren'Py\renpy-8.3.4-sdk\renpy\main.py", line 148, in run
    renpy.execution.run_context(True)
  File "C:\Users\Corv1122\Downloads\Ren'Py\renpy-8.3.4-sdk\renpy\execution.py", line 958, in run_context
    context.run()
  File "game/script.rpy", line 252, in script call
    call setup_icons()
  File "game/script.rpy", line 44, in script
    call screen Match_Three
  File "C:\Users\Corv1122\Downloads\Ren'Py\renpy-8.3.4-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'win_screen'.

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.4.24120703
EcoCity Match 0.1
Mon Apr 14 11:04:54 2025
